packages/core/src/index.ts

```diff
@@ -2,38 +2,40 @@
 // https://github.com/microsoft/fluentui/issues/20694
 import {
   border,
-  borderLeft,
   borderBottom,
-  borderRight,
-  borderTop,
   borderColor,
-  borderStyle,
+  borderLeft,
   borderRadius,
+  borderRight,
+  borderStyle,
+  borderTop,
   borderWidth,
+  flex,
   gap,
-  margin,
-  padding,
-  overflow,
   inset,
+  margin,
   outline,
+  overflow,
+  padding,
 } from './shorthands/index';
 
 export const shorthands = {
   border,
-  borderLeft,
   borderBottom,
-  borderRight,
-  borderTop,
   borderColor,
-  borderStyle,
+  borderLeft,
   borderRadius,
+  borderRight,
+  borderStyle,
+  borderTop,
   borderWidth,
+  flex,
   gap,
-  margin,
-  padding,
-  overflow,
   inset,
+  margin,
   outline,
```

Comment:
This PR should not reorganize export order, this has nothing to do with `flex` shorthand implementation.